lib/src/screens/detail_screen/detail_screen.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:syncfusion_flutter_charts/sparkcharts.dart';
```

Comment:
is this an external package? I remember that I have noticed all of you should let me know what library you want to add to the app, didn't I?

Reply:
Yes, you have already told it before. Sorry for the unmentioned at the beggining of the assignment. So can we use it to draw the chart on our app?